cms/djangoapps/contentstore/views/course.py

```diff
@@ -1459,32 +1459,109 @@ def _get_usage_info(store, course, split_tests):
 
             validation_summary = split_test.general_validation_message()
             usage_info[split_test.user_partition_id].append({
-                'label': '{} / {}'.format(unit.display_name, split_test.display_name),
+                'label': u"{} / {}".format(unit.display_name, split_test.display_name),
                 'url': unit_url,
                 'validation': validation_summary.to_json() if validation_summary else None,
             })
         return usage_info
 
     @staticmethod
+    def get_content_groups_usage_info(store, course):
+        """
+        Get usage information for content Group Configurations.
+        """
+        modules = store.get_items(course.id, settings={'group_access': {'$exists': True}})
+
+        return GroupConfiguration._get_content_groups_usage_info(store, course, modules)
+
+    @staticmethod
+    def _get_content_groups_usage_info(store, course, modules):
+        """
+        Returns all units names and their urls.
+
+        Returns:
+        {'group_id':
+            [
+                {
+                    'label': 'Unit 1 / Problem 1',
+                    'url': 'url_to_unit_1'
+                },
+                {
+                    'label': 'Unit 2 / Problem 2',
+                    'url': 'url_to_unit_2'
+                }
+            ],
+        }
+        """
+        usage_info = {}
+        for module in modules:
+            if hasattr(module, 'group_access') and module.group_access:
+                (__, group_ids), = module.group_access.items()
+                for group_id in group_ids:
+                    if group_id not in usage_info:
+                        usage_info[group_id] = []
+
+                    unit_location = store.get_parent_location(module.location)
```

Comment:
you should be able to do module.get_parent() now, and that should actually be much more efficient than using modulestore calls.  In general I'd like to encourage using get_parent / get_children to move around within the structure of a course - is there any reason this can't be done here?  you may not need to pass a `store` instance around at all.